mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -1739,14 +1852,17 @@ func ResourceContainerCluster() *schema.Resource {
 						"enable_private_endpoint": {
 							Type:             schema.TypeBool,
 							Optional:         true,
+							Deprecated:       "use control_plane_endpoints_config.ip_endpoints_config.enable_public_endpoint instead, and note the flipped semantics",
```

Comment:
Similar to above, can we add a test case that migrates between these settings? Ideally in a separate case that does not have a value for master_authorized_networks_config rather than combined into one sample.

Reply:
Done as part of TestAccContainerCluster_migratePrivateClusterConfig